homeassistant/components/sensibo/climate.py

```diff
@@ -314,11 +314,14 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
         if "targetTemperature" not in self.device_data.active_features:
             raise HomeAssistantError(
-                "Current mode doesn't support setting Target Temperature"
```

Comment:
The message shouldn't have been removed from the error object, as this is used for logging and such.